recipes/tmtk/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "tmtk" %}
+{% set version = "0.4.4.dev3" %}
+{% set sha256 = "d94600bd60b120ae96ef9e6fad5019114836bf6da0bc181113ab7433a2d04f75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```

Comment:
Are all of these packages required to `build` the recipe? Or are they runtime reqs except for `python` and `pip`?

Also `python` is listed twice under `build`

Reply:
Thank you for the thorough review. Some obvious things to fix here.

Regarding the build requirements I was surprised it needs them actually. The build had failed  with a `ImportError: No module named 'pandas'` (build log: https://circleci.com/gh/conda-forge/staged-recipes/24591), therefore I added them.

Perhaps this is due to the install script automatically installing a Jupyter notebook extension. Any suggestions for a best practice?

Reply:
Changed the setup script, the run requirements are no longer needed to build. Thanks.